src/components/idleMembers/card/index.tsx

```diff
@@ -1,13 +1,13 @@
-import { FC } from 'react';
+import { FC, useState } from 'react';
 import classNames from '@/components/idleMembers/card/card.module.scss';
 
 type Props = {
   idleMemberUserName: string
 }
 
 const Card: FC<Props> = ({ idleMemberUserName }) => {
-  const imageGenerator = (name: string) => `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${name}/img.png`;
-
+  const [assigneeProfilePic, setAssigneeProfilePic] = useState(`${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${idleMemberUserName}/img.png`);
+  const assigneeImageOnError = () => setAssigneeProfilePic('dummyProfile.png');
```

Comment:
Can you add dummyProfile image in a separate variable? And use that variable over here.

Reply:
Also, make another variable for URL inside of useState.